images/macos/provision/bootstrap-provisioner/installNewProvisioner.sh

```diff
@@ -6,11 +6,11 @@ ScriptName="$4"
 ScriptParam="$5"
 Username="$6"
 
-export PATH=/usr/bin:/usr/sbin:/usr/local/bin:/bin:/sbin
+export PATH=/usr/bin:/usr/sbin:/usr/local/bin:/bin:/sbin:/opt/homebrew/bin
```

Comment:
@finaroux I am wondering if we should make PATH arch-conditional and add /opt/homebrew/bin only if `$(arch) == arm64`?

Reply:
If it is what you are doing in your other PR that makes sense. I will update it.